lib/serviceProperties.js

```diff
@@ -10,13 +12,28 @@ function convertToKeyValueFromSpec(spec, prop) {
   }
 }
 
+function getExecFromSpec(spec) {
+  const nodeVersion = getValueFromSpec(spec, 'nodeVersion');
+  if (!nodeVersion) {
+    return '/usr/bin/npm';
+  }
+
+  if (semver.minVersion(nodeVersion).major >= 22) {
+    return '/usr/bin/node --run';
+  }
+
+  return 'usr/bin/npm';
```

Comment:
Looks, by the other return values to need a `/` at the start
```suggestion
  return '/usr/bin/npm';
```